src/pages/contributing/documentation.mdx

```diff
@@ -768,30 +817,30 @@ Provide details about the clickable areas of the component.
 ### Loading
 
 <!--
-If applicable, use this section to cross reference to the specific loading type in the [Loading component](https://www.carbondesignsystem.com/patterns/loading-pattern) and/or the [Loading pattern](https://www.carbondesignsystem.com/patterns/loading-pattern/).
+If applicable, use this section to cross reference to the specific loading type in the [Loading component](/components/loading/usage) and/or the [Loading pattern](/patterns/loading-pattern).
```

Comment:
Should the "type" in this sentence be "variant"?

Reply:
Yes indeed! Thank you.